plan/physical_plan_builder.go

```diff
@@ -1048,6 +1048,58 @@ func (p *Distinct) convert2PhysicalPlan(prop *requiredProperty) (*physicalPlanIn
 	return info, nil
 }
 
+// convert2PhysicalPlan implements the LogicalPlan convert2PhysicalPlan interface.
+func (p *Analyze) convert2PhysicalPlan(prop *requiredProperty) (*physicalPlanInfo, error) {
+	info, err := p.getPlanInfo(prop)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	if info != nil {
+		return info, nil
+	}
+	var childInfos []*physicalPlanInfo
+	for _, ind := range p.IndOffsets {
+		var columns []*model.ColumnInfo
+		tblInfo := p.Table.TableInfo
+		for _, indCol := range tblInfo.Indices[ind].Columns {
+			for _, col := range tblInfo.Columns {
+				if col.Name.L == indCol.Name.L {
+					columns = append(columns, col)
+					break
+				}
+			}
+		}
+		is := &PhysicalIndexScan{
+			Index:               tblInfo.Indices[ind],
+			Table:               tblInfo,
+			Columns:             columns,
+			TableAsName:         &p.Table.Name,
+			OutOfOrder:          true,
```

Comment:
Seems like a mistake. It was `true` because I refer to the settings in `convert2IndexScan`. Thanks.